### PR TITLE
fix: Add missing PathUtils to DOMRenderers

### PR DESCRIPTION
**Pull request by alexanderGugel**

### CODE REVIEW

File: dom-renderers/DOMRenderer.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ var ElementCache = require('./ElementCache');
 var math = require('./Math');
 var vendorPrefix = require('../utilities/vendorPrefix');
 var eventMap = require('./events/EventMap');
+var PathUtils = require('../core/PathUtils');
```
Comment (DnMllr) [inline dom-renderers/DOMRenderer.js]: It's not named PathUtils, it's just named Path. Though the variable name is PathUtils pretty universally so the file should also be renamed. You could either rename the file and change the require statements or you could change the require statement here.

Comment (alexanderGugel) [inline dom-renderers/DOMRenderer.js]: Did you rename that or am I about to turn crazy?

**Comment by alexanderGugel** [comment]
@DnMllr fixed

**Comment by alexanderGugel** [comment]
seems like you fixed that

**Pull request closed without merging**